Changelog.md

```diff
@@ -2,11 +2,11 @@
 ### Added
 - Progress indicators for app install/upgrade
 - Enables the apps to show an additional documentation card in their About pane
+- About pane with version, documentation and license information
 - Windows x64 support
-### Updated
-- Replaced underlying communication with devices by integrating nrf-device-lib, to enhance reliability
```

Comment:
Why did you remove this? I put it (rephrased) back in again because I consider it to be helpful.